proto/v1/skaffold.proto

```diff
@@ -690,6 +690,22 @@ enum StatusCode {
     TEST_CST_USER_ERR = 1102;
     // Unable to docker pull image
     TEST_IMG_PULL_ERR = 1103;
+    // Error running custom test
```

Comment:
need to move these to `proto/enums/enums.proto` after another recent PR got merged.

Reply:
in V2?